### ISSUE TITLE
prepare-wmt14en2de.sh

### ISSUE THREAD

**Issue by zhao1402072392** [pytorch/fairseq#1618]
this .sh file is for getting WMT17en-de? not WMT14en-de,right?,and what  the valid dataset and test test dataset are?

**Comment by myleott** [comment]
Correct, by default it preprocesses the WMT'17 training data.
The valid set is newstest2013: https://github.com/pytorch/fairseq/blob/master/examples/translation/prepare-wmt14en2de.sh#L60
The test set is newstest2014: https://github.com/pytorch/fairseq/blob/master/examples/translation/prepare-wmt14en2de.sh#L106

**Comment by zhao1402072392** [comment]
thanks for your answer.so how to revise the code to gain the WMT14 trainig data,please:) @myleott

**Comment by myleott** [comment]
Just pass the `--icml17` flag to the prepare script and it should produce WMT14 training data. This is also explained a bit here: https://github.com/pytorch/fairseq/blob/master/examples/translation/README.md#wmt14-english-to-german-convolutional

**Comment by mrqorib** [comment]
> Correct, by default it preprocesses the WMT'17 training data.
> The valid set is newstest2013: https://github.com/pytorch/fairseq/blob/master/examples/translation/prepare-wmt14en2de.sh#L60
> The test set is newstest2014: https://github.com/pytorch/fairseq/blob/master/examples/translation/prepare-wmt14en2de.sh#L106

Hi sorry I just want to ask, according to https://www.statmt.org/wmt17/translation-task.html , isn't this dataset lacking Rapid corpus of EU press releases dataset to be called WMT 2017?
Thank you!
